<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guradiansignup</class>
 <widget class="QDialog" name="guradiansignup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{background-image: url(:/img/img/1772226 (1).jpg);}
QMessageBox{
    font: 87 10pt &quot;Segoe UI Black&quot;;
}
QPushButton:pressed{
	background-color: rgb(255, 212, 39);
   border-radius:6px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>329</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLineEdit" name="lineEdit_phone_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_inst_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_class_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="styleSheet">
       <string notr="true">background:black;
color:rgb(255, 215, 52);
font: 87 10pt &quot;Segoe UI Black&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
text-decoration: underline;
font:87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Sign Up as a Guardian</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>608</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>National Curriculam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>English Medium</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>251</y>
     <width>329</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLineEdit" name="lineEdit_name_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_city_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_group_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pass_2">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_signup">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background:black;
color:rgb(255, 197, 20);
font: 87 13pt &quot;Segoe UI Black&quot;;
border:black;
}
QPushButton:pressed{
	
	background-color: rgb(255, 170, 0);
   border-radius:6px;
}
QPushButton:hover{
	
	background-color:rgb(170, 0, 0);
   border-radius:6px;
}
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>251</y>
     <width>145</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_name_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_city_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_group_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pass_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>172</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_phone_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_ins_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Institution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_class_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Class/Dept.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/img/icon.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>252</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_un">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>327</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background:black;
color:rgb(255, 197, 20);
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_un">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{image: url(:/img/img/backbutoonyblack.jpg);
border:yellow}
QPushButton:pressed{
	
	background-color: rgb(255, 170, 0);
   border-radius:6px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
